lib/rules/boolean-prop-naming.js

```diff
@@ -76,7 +76,10 @@ module.exports = {
       if (node.value.property) {
         const name = node.value.property.name;
         if (name === 'isRequired') {
-          return node.value.object.property.name;
+          if (node.value.object.property) {
```

Comment:
what if `node.value.object` is falsy?

Reply:
I couldn't find a case where node.value has a property but no object. But I added a guard anyways.